.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,7 @@
+npm install && make install-env
+npm run prepare
+make install-gum
+poetry config virtualenvs.in-project true
+make install-web
```

Comment:
Let's break these into groups and add some comments as to make it easier for newcomers.

Reply:
Solved in 1dad3d9